src/PrebidMobile/PrebidMobile/BidManager.swift

```diff
@@ -137,11 +137,11 @@ import Foundation
                     return ([:],ResultCode.prebidDemandNoBids)
                 }
             } else {
-                if(errorString.contains("Stored Imp with ID")){
+                if(errorString.contains("Stored Imp with ID") || errorString.contains("No stored imp found")){
                     return ([:],ResultCode.prebidInvalidConfigId)
-                } else if(errorString.contains("Stored Request with ID")) {
+                } else if(errorString.contains("Stored Request with ID") || errorString.contains("No stored request found")) {
                     return ([:],ResultCode.prebidInvalidAccountId)
-                } else if((errorString.contains("Invalid request: Request imp[0].banner.format")) || (errorString.contains("Unable to set interstitial size list"))){
+                } else if((errorString.contains("Invalid request: Request imp[0].banner.format")) || errorString.contains("Request imp[0].banner.format") || (errorString.contains("Unable to set interstitial size list"))){
```

Comment:
`(errorString.contains("Invalid request: Request imp[0].banner.format")) || errorString.contains("Request imp[0].banner.format")`

cant it be only this
`errorString.contains("Request imp[0].banner.format")`???

Reply:
I left the first string on purpose to have two different variants that allow in future just to change a corresponding string or, moreover, refactor this code by separating on AppNexus and RubiconProject strings.